app/views/includes/summary/previouslyConnected.scala.html

```diff
@@ -19,9 +19,6 @@
 
 @(notConnectedSummary: NotConnectedSummary)(implicit requestHeader: RequestHeader, messages: Messages)
 
-<h2 class="govuk-heading-m">@messages("notConnected.previouslyConnected")</h2>
-<dl class="govuk-summary-list">
```

Comment:
Did you mean to remove the dl tags?

Reply:
I removed the DL from one as there was an awkward gap between the first section and the second section. Removing the dl tag brought them together. I can change it to something else using the dl tags if this isn't best practice?